spec/polymorphic_integer_type_spec.rb

```diff
@@ -371,4 +371,13 @@ class InlineDrink2 < ActiveRecord::Base
       expect(link.normal_target_type).to eq("InlineDrink2")
     end
   end
+
+  context "when using other reflection" do
```

Comment:
What is `other`? Can we be more specific?

Reply:
I specify one example underneath in `it` aka `ActiveRecord::Reflection::ThroughReflection`